stable/traefik/templates/service.yaml

```diff
@@ -12,12 +12,10 @@ metadata:
     {{ $key }}: {{ $value | quote }}
   {{- end }}
   {{- end }}
+{{- if .Values.service.annotations }}
   annotations:
-  {{- if .Values.service }}
-  {{- range $key, $value := .Values.service.annotations }}
-    {{ $key }}: {{ $value | quote }}
-  {{- end }}
-  {{- end }}
+{{ toYaml .Values.service.annotations | indent 4 }}
```

Comment:
This changes the quoting from past versions...
`testing: 10` will not behave the same with this PR...

Thoughts?

Reply:
Quote shouldn't be required since rather than manually building up yaml we're relying on the internal `toYaml` function now.  If the end user needs to specifical quote the "value" it can be quoted in the values.yaml/etc right?

Reply:
The problem is with parsing int and bools.
In the current version, all values are quoted to ensure they will parse properly as strings.
You can run into issues with using annotations that don't parse to strings such as:
`traefik.ingress.kubernetes.io/preserve-host: "true"`

Currently, the user will have:
`traefik.ingress.kubernetes.io/preserve-host: true`, and the parsed value will be quoted.
With `toYaml`, the `true` will not be quoted, and will cause parsing issues.

Reply:
That's fair,

I'm fine with reverting back to the key/vault method but I'd like to move `{{- if .Values.service }}` above `annotations:` so in the case of no annotations we don't leave behind a empty `annotations:` with nothing following.  The blank field was causing yaml validation failures.  If that works for you I'll update the code @dtomcej 

Thanks,